sources/packages/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -52,3 +52,36 @@ export interface ProgramOfferingDto extends ProgramOfferingBaseDTO {
 export interface ProgramOfferingDetailsDto {
   studyStartDate?: Date;
 }
+
+/**
+ * Transformation util for Program Offering.
+ * @param offering
+ * @returns ProgramOfferingDto
+ */
+export const transformToProgramOfferingDto = (
+  offering: ProgramOfferingModel,
+): ProgramOfferingDto => {
+  return {
+    id: offering.id,
+    offeringName: offering.name,
+    studyStartDate: offering.studyStartDate,
+    studyEndDate: offering.studyEndDate,
+    actualTuitionCosts: offering.actualTuitionCosts,
+    programRelatedCosts: offering.programRelatedCosts,
+    mandatoryFees: offering.mandatoryFees,
+    exceptionalExpenses: offering.exceptionalExpenses,
+    tuitionRemittanceRequestedAmount: offering.tuitionRemittanceRequestedAmount,
+    offeringDelivered: offering.offeringDelivered,
+    lacksStudyDates: offering.lacksStudyDates,
+    lacksStudyBreaks: offering.lacksStudyBreaks,
+    lacksFixedCosts: offering.lacksFixedCosts,
+    tuitionRemittanceRequested: offering.tuitionRemittanceRequested,
+    offeringIntensity: offering.offeringIntensity,
+    yearOfStudy: offering.yearOfStudy,
+    showYearOfStudy: offering.showYearOfStudy,
+    hasOfferingWILComponent: offering.hasOfferingWILComponent,
+    offeringWILType: offering.offeringWILType,
+    studyBreaks: offering.studyBreaks,
```

Comment:
for studyBreaks, please see previous comments I made on lacksStudyBreaks

Reply:
added comment above